feat(nuxt): allow keeping fallback for `NuxtClientFallback` (#20336)
nuxt/nuxt: docs/3.api/2.components/1.nuxt-client-fallback.md

  - **default**: `div`

> - **placeholder** | **fallback**: Specify fallback content to be rendered if the slot fails to render.

Reviewer: My bad I confused the props names

  - **type**: `string`

> - **strategy**: Specify a strategy to adopt once in client-side. By default it tries to render the slot if it fails in client-side but your can choose to keep the fallback content by setting this prop to `keep-fallback`

Reviewer 2: ```suggestion
- **strategy**: Specify a strategy to adopt once client-side. By default it tries to re-render the slot on the client but you can choose to keep the fallback content by setting this prop to `keep-fallback`
```

>   - **type**: `string`

Reviewer: Maybe we could say something like 
```
`default | keep-fallback`
```

>   - **type**: `'default'|'fallback'`

Reviewer 2: ```suggestion
  - **type**: `'default' | 'fallback'`
```
